asmcomp/cmm_helpers.mli

```diff
@@ -555,9 +559,12 @@ val bigstring_load :
 
 (** Arrays *)
 
-(** Array access. Args: array, index *)
-val arrayref_unsafe : Lambda.array_kind -> binary_primitive
-val arrayref_safe : Lambda.array_kind -> binary_primitive
+(** Array access.  The [alloc_mode] is used only if the [array_kind] is
+    [Pfloatarray]; the resulting [float] will then be allocated at the specified
+    mode.
```

Comment:
Again, change the type so the condition is statically checked and doesn't need a comment.

Reply:
Done in 2f3811e0c669a59bd1f44390c63f98d48abcf88d; see above